iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/StaticVarCompensatorAdapter.java

```diff
@@ -43,28 +43,48 @@ public StaticVarCompensator setBmax(final double bMax) {
         return this;
     }
 
+    @Override
+    public double getVoltageSetpoint() {
+        return getDelegate().getVoltageSetpoint();
+    }
+
     @Override
     public double getVoltageSetPoint() {
-        return getDelegate().getVoltageSetPoint();
+        return getVoltageSetpoint();
```

Comment:
Same remark as above, you can delete the implementations of deprecated methods if they have default implementations in the interface